continuedev/continue
# feat: include recently + open files in codebase search

## Description

Includes recently + open files in codebase search.

## Checklist


## Screenshots

[ "For visual changes, include screenshots." ]

## Testing

- Open `core/context/retrieval/pipelines/RerankerRetrievalPipeline.ts`
- Comment out the other retrieval steps
```ts
retrievalResults.push(
    ...recentlyEditedFilesChunks,
    // ...ftsChunks,
    // ...embeddingsChunks,
  );
```
- Tab through a few files, submit a prompt using codebase 
- Confirm that those files are used as context
- Reload the window
- Submit a prompt using codebase, confirm the open tabs are used as context
